.github/workflows/cpu-inference.yml

```diff
@@ -69,7 +82,11 @@ jobs:
       - name: Unit tests
         run: |
           source oneCCL/build/_install/env/setvars.sh
+          # check whether the environment is properly setup
+          python -c "import torch;import intel_extension_for_pytorch as ipex;import oneccl_bindings_for_pytorch;print('done')"
```

Comment:
Might be worth moving these up one section after the call to `pip list`?

Reply:
These checks are moved up to the `pip list` section